cumulus/polkadot-parachain/src/chain_spec/collectives.rs

```diff
@@ -22,35 +22,33 @@ use parachains_common::{AccountId, AuraId, Balance as CollectivesBalance};
 use sc_service::ChainType;
 use sp_core::sr25519;
 
-pub type CollectivesPolkadotChainSpec =
-	sc_service::GenericChainSpec<collectives_polkadot_runtime::RuntimeGenesisConfig, Extensions>;
+pub type CollectivesWestendChainSpec =
```

Comment:
Same, not 100% sure, but we might need the node to know about both Collectives Westend and Polkadot.